include/onnxruntime/core/common/exceptions.h

```diff
@@ -58,6 +65,9 @@ class OnnxRuntimeException : public std::exception {
     what_ = ss.str();
   }
 
+  const StatusCode GetStatusCode() const noexcept {
```

Comment:
>GetStatusCode [](start = 19, length = 13)

i just double checked the code, and it looks like Status has a default constructor and it appears to do all the right things.

Reply:
I changed to return the status itself, instead of just the status code.


---
In reply to: [355751619](https://github.com/microsoft/onnxruntime/pull/2596#discussion_r355751619) [](ancestors = 355751619)